src/helpers/strings-de.ts

```diff
@@ -149,7 +149,6 @@ let strings: { [key: string]: string } = {
     "systolic-average": "Durchschnittlicher systolischer Blutdruck",
     "diastolic-average": "Durchschnittlicher diastolischer Blutdruck",
     "highest-systolic": "Höchster systolischer Blutdruck",
-    "lowest-diastolic": "Niedrigster diastolischer Blutdruck",
```

Comment:
Can we update these instead of deleting?

Reply:
I found the spanish translation file was missing all BP labels. Updated to include those